proposed/extended-coding-style-guide.md

```diff
@@ -391,6 +391,8 @@ The `var` keyword MUST NOT be used to declare a property.
 
 There MUST NOT be more than one property declared per statement.
 
+There MUST be one space after and before the equal sign.
```

Comment:
I agree that this should be the norm, as I see this being practiced everywhere in PSR 1 and 2 code examples, but I don't think we should restrict this rule to the assignment operator only, but to all operators. Also, I think there is the odd one, the equals sign used in declare statements, since it's now widely more popular due to the strict type use in PHP 7.